.releaserc.json

```diff
@@ -0,0 +1,140 @@
+{
+   "branches":[
+      "main",
+      {
+         "name":"test",
```

Comment:
Let's change to `develop` and it will be possible to make pre-releases from `develop` branch